### PR TITLE
Create 2016-04-19-shoutouts-week13.md

### LINKED ISSUE #88: shout out blog post
hey @sjnorth, thanks to @finnp’s work, we know have a script that gets more data for us to give shout outs. Here’s a summary for contirbutions since last shout out, can you work with that? It says who contributed for the first time, who reviewed, and you can give some extra shout outs based on contributions, like Finn’s work on the internal architecture migration and Cecily on the TGIFs. Let me know if you have any farther question

[Charlotteis](https://github.com/Charlotteis) worked on 7 repositories.
- All-time total: 62 merged PRs.
- landed [fix: typo and removing the use of 'simple'](https://github.com/hoodiehq/hoodie-camp-tutorial/pull/9)
- merged 3 pull requests
- reviewed 3 pull requests
- commented 11 times

[HipsterBrown](https://github.com/HipsterBrown) worked on 8 repositories.
- All-time total: 26 merged PRs.
- merged 6 pull requests
- reviewed 19 pull requests
- commented 22 times

[HopeMobile](https://github.com/HopeMobile) worked on [hoodiehq/hoodie-client](https://github.com/hoodiehq/hoodie-client)
- All-time total: 1 merged PRs.
- First PR on the hoodie project!
- landed [docs(README): fixed git clone URL](https://github.com/hoodiehq/hoodie-client/pull/69)
- commented on [hoodiehq/hoodie-client#68](https://github.com/hoodiehq/hoodie-client/issues/68#issuecomment-204760360)

[KaLaUnlimited](https://github.com/KaLaUnlimited) worked on [hoodiehq/hoodie-account-server](https://github.com/hoodiehq/hoodie-account-server)
- All-time total: 0 merged PRs
- commented on [hoodiehq/hoodie-account-server#120](https://github.com/hoodiehq/hoodie-account-server/issues/120#issuecomment-204760466)

[NeekyRabbit](https://github.com/NeekyRabbit) worked on [hoodiehq/editorial](https://github.com/hoodiehq/editorial)
- All-time total: 0 merged PRs.
- commented on [hoodiehq/editorial#64](https://github.com/hoodiehq/editorial/issues/64#issuecomment-208770917)

[SonjaHeinen](https://github.com/SonjaHeinen) worked on [hoodiehq/hood.ie](https://github.com/hoodiehq/hood.ie)
- All-time total: 0 merged PRs.
- commented 3 times

[alisabzevari](https://github.com/alisabzevari) worked on [hoodiehq/hoodie-account-server](https://github.com/hoodiehq/hoodie-account-server)
- All-time total: 1 merged PRs.
- commented on [hoodiehq/hoodie-account-server#126](https://github.com/hoodiehq/hoodie-account-server/pull/126#issuecomment-202142090)

[antoinelyset](https://github.com/antoinelyset) worked on [hoodiehq/hoodie-connection-status](https://github.com/hoodiehq/hoodie-connection-status)
- All-time total: 1 merged PRs.
- commented on [hoodiehq/hoodie-connection-status#48](https://github.com/hoodiehq/hoodie-connection-status/issues/48#issuecomment-205715171)

[artivilla](https://github.com/artivilla) worked on [hoodiehq/hoodie-account-server](https://github.com/hoodiehq/hoodie-account-server)
- All-time total: 1 merged PRs.
- commented on [hoodiehq/hoodie-account-server#80](https://github.com/hoodiehq/hoodie-account-server/issues/80#issuecomment-201704508)

[boennemann](https://github.com/boennemann) worked on [hoodiehq/hoodie-account-server](https://github.com/hoodiehq/hoodie-account-server)
- All-time total: 68 merged PRs.
- merged 1 pull requests
- reviewed 1 pull requests
- commented on [hoodiehq/hoodie-account-server#144](https://github.com/hoodiehq/hoodie-account-server/pull/144#issuecomment-206455472)

[bryanbierce](https://github.com/bryanbierce) worked on [hoodiehq/hoodie-account-server](https://github.com/hoodiehq/hoodie-account-server)
- All-time total: 0 merged PRs.
- commented 3 times

[caityc](https://github.com/caityc) worked on [hoodiehq/hoodie-account](https://github.com/hoodiehq/hoodie-account)
- All-time total: 1 merged PRs.
- First PR on the hoodie project!
- landed [docs(README): fixed local server URL](https://github.com/hoodiehq/hoodie-account/pull/6)
- commented 2 times

[chrifpa](https://github.com/chrifpa) worked on [hoodiehq/hoodie-account-client](https://github.com/hoodiehq/hoodie-account-client)
- All-time total: 2 merged PRs.
- First PR on the hoodie project!
- landed [#83: Bind null in object constructor](https://github.com/hoodiehq/hoodie-account-client/pull/87)
- landed [#66 test: 100% coverage](https://github.com/hoodiehq/hoodie-account-client/pull/78)
- commented 7 times

[chrisdemars](https://github.com/chrisdemars) worked on [hoodiehq/hood.ie](https://github.com/hoodiehq/hood.ie)
- All-time total: 0 merged PRs.
- commented 5 times

[christophwitzko](https://github.com/christophwitzko) worked on [hoodiehq/hoodie-account-client](https://github.com/hoodiehq/hoodie-account-client)
- All-time total: 15 merged PRs.
- merged 1 pull requests
- reviewed 1 pull requests
- commented on [hoodiehq/hoodie-account-client#78](https://github.com/hoodiehq/hoodie-account-client/pull/78#issuecomment-201752803)

[coderbyheart](https://github.com/coderbyheart) worked on [hoodiehq/pouchdb-hoodie-sync](https://github.com/hoodiehq/pouchdb-hoodie-sync)
- All-time total: 14 merged PRs.
- landed [docs(LICENSE): unify license accross all active repositories](https://github.com/hoodiehq/pouchdb-hoodie-sync/pull/63)

[codyzu](https://github.com/codyzu) worked on [hoodiehq/hoodie-account-server](https://github.com/hoodiehq/hoodie-account-server)
- All-time total: 3 merged PRs.
- First PR on the hoodie project!
- landed [Issue/136](https://github.com/hoodiehq/hoodie-account-server/pull/137)
- landed [fix(routes): POST /accounts?include=foobar -> 400](https://github.com/hoodiehq/hoodie-account-server/pull/132)
- landed [Issue/37 fix and re-enable coverage and badge](https://github.com/hoodiehq/hoodie-account-server/pull/146)
- commented 7 times

[coveralls](https://github.com/coveralls) worked on [hoodiehq/hoodie-connection-status](https://github.com/hoodiehq/hoodie-connection-status)
- All-time total: 0 merged PRs.
- commented on [hoodiehq/hoodie-connection-status#52](https://github.com/hoodiehq/hoodie-connection-status/pull/52#issuecomment-203997480)

[dar5hak](https://github.com/dar5hak) worked on [hoodiehq/hoodie-server](https://github.com/hoodiehq/hoodie-server)
- All-time total: 1 merged PRs.
- First PR on the hoodie project!
- landed [docs: fix typo in README](https://github.com/hoodiehq/hoodie-server/pull/481)

[doublerebel](https://github.com/doublerebel) worked on [hoodiehq/hoodie-server](https://github.com/hoodiehq/hoodie-server)
- All-time total: 0 merged PRs.
- commented on [hoodiehq/hoodie-server#171](https://github.com/hoodiehq/hoodie-server/issues/171#issuecomment-205679129)

[espy](https://github.com/espy) worked on [hoodiehq/editorial](https://github.com/hoodiehq/editorial)
- All-time total: 20 merged PRs.
- commented 2 times

[finnp](https://github.com/finnp) worked on 11 repositories.
- All-time total: 18 merged PRs.
- landed [Fix links that were changed in new architecture](https://github.com/hoodiehq/hoodie/pull/460)
- landed [Migrate to new Architecture](https://github.com/hoodiehq/hoodie-server/pull/485)
- landed [Migrate to new hoodie architecture](https://github.com/hoodiehq/hoodie-client/pull/65)
- landed [test: move account-test to @hoodie/account](https://github.com/hoodiehq/hoodie-app-tracker/pull/52)
- landed [refactor: change dependencies to @hoodie](https://github.com/hoodiehq/hoodie-account/pull/1)
- landed [Rename module and repo](https://github.com/hoodiehq/hoodie-account-server/pull/131)
- landed [docs(README): fix badge links for new repo name](https://github.com/hoodiehq/hoodie-account-server/pull/135)
- landed [Rename module and repo](https://github.com/hoodiehq/hoodie-account-client/pull/86)
- merged 4 pull requests
- reviewed 7 pull requests
- commented 17 times

[gitsemere](https://github.com/gitsemere) worked on [hoodiehq/hoodie-client-task](https://github.com/hoodiehq/hoodie-client-task)
- All-time total: 1 merged PRs.
- First PR on the hoodie project!
- landed [Changed Url](https://github.com/hoodiehq/hoodie-client-task/pull/27)
- commented on [hoodiehq/hoodie-client-task#26](https://github.com/hoodiehq/hoodie-client-task/issues/26#issuecomment-204765176)

[gr2m](https://github.com/gr2m) worked on 21 repositories.
- All-time total: 222 merged PRs.
- landed [Nick’s post: make first paragraph a paragraph :)](https://github.com/hoodiehq/hood.ie/pull/269)
- landed [clear items on signout](https://github.com/hoodiehq/hoodie-app-tracker/pull/53)
- landed [unskip ".findAll() objects after signin"](https://github.com/hoodiehq/hoodie-app-tracker/pull/54)
- landed [cross-module initialisations](https://github.com/hoodiehq/hoodie-client/pull/63)
- landed [reset fixes](https://github.com/hoodiehq/hoodie-store-client/pull/97)
- merged 28 pull requests
- reviewed 35 pull requests
- commented 167 times

[greenkeeperio-bot](https://github.com/greenkeeperio-bot) worked on 9 repositories.
- All-time total: 121 merged PRs.
- landed [Update nock to version 8.0.0 🚀](https://github.com/hoodiehq/hoodie-server/pull/486)
- landed [Update nock to version 8.0.0 🚀](https://github.com/hoodiehq/hoodie-store-server/pull/38)
- landed [Update nock to version 8.0.0 🚀](https://github.com/hoodiehq/hoodie-connection-status/pull/53)
- landed [Update simple-mock to version 0.7.0 🚀](https://github.com/hoodiehq/hoodie-client/pull/67)
- landed [Update simple-mock to version 0.7.0 🚀](https://github.com/hoodiehq/hoodie-store-client/pull/93)
- landed [Update simple-mock to version 0.7.0 🚀](https://github.com/hoodiehq/hoodie-log/pull/34)
- landed [nock@7.7.1 breaks build 🚨](https://github.com/hoodiehq/hoodie-account-server/pull/133)
- landed [Update standard to version 6.0.8 🚀](https://github.com/hoodiehq/hoodie-plugin-users/pull/72)
- landed [Update slackin to version 0.8.3 🚀](https://github.com/hoodiehq/slackin/pull/13)

[hintsa](https://github.com/hintsa) worked on 3 repositories.
- All-time total: 1 merged PRs.
- First PR on the hoodie project!
- landed [effect less chane](https://github.com/hoodiehq/documentation/pull/205)
- commented 3 times

[hyperperforator](https://github.com/hyperperforator) worked on [hoodiehq/hoodie-account-server](https://github.com/hoodiehq/hoodie-account-server)
- All-time total: 1 merged PRs.
- First PR on the hoodie project!
- landed [404 should be 401 on /session](https://github.com/hoodiehq/hoodie-account-server/pull/142)
- commented 6 times

[hzoo](https://github.com/hzoo) worked on [hoodiehq/hoodie-account-server](https://github.com/hoodiehq/hoodie-account-server)
- All-time total: 0 merged PRs.
- commented on [hoodiehq/hoodie-account-server#141](https://github.com/hoodiehq/hoodie-account-server/pull/141#issuecomment-204771072)

[inator](https://github.com/inator) worked on [hoodiehq/hoodie](https://github.com/hoodiehq/hoodie)
- All-time total: 1 merged PRs.
- commented on [hoodiehq/hoodie#458](https://github.com/hoodiehq/hoodie/issues/458#issuecomment-204477836)

[janl](https://github.com/janl) worked on 3 repositories.
- All-time total: 58 merged PRs.
- merged 1 pull requests
- reviewed 1 pull requests
- commented 14 times

[jkaplon](https://github.com/jkaplon) worked on [hoodiehq/faq](https://github.com/hoodiehq/faq)
- All-time total: 0 merged PRs.
- commented on [hoodiehq/faq#129](https://github.com/hoodiehq/faq/issues/129#issuecomment-201415528)

[jo-porter](https://github.com/jo-porter) worked on [hoodiehq/editorial](https://github.com/hoodiehq/editorial)
- All-time total: 1 merged PRs.
- commented on [hoodiehq/editorial#68](https://github.com/hoodiehq/editorial/issues/68#issuecomment-207482944)

[jsimplicio](https://github.com/jsimplicio) worked on [hoodiehq/editorial](https://github.com/hoodiehq/editorial)
- All-time total: 7 merged PRs.
- commented 2 times

[jywarren](https://github.com/jywarren) worked on [hoodiehq/hoodie-app-tracker](https://github.com/hoodiehq/hoodie-app-tracker)
- All-time total: 0 merged PRs.
- commented 2 times

[kazekagegaara](https://github.com/kazekagegaara) worked on [hoodiehq/pouchdb-admins](https://github.com/hoodiehq/pouchdb-admins)
- All-time total: 2 merged PRs.
- landed [issue 8 fix, changed from lodash.clone to lodash](https://github.com/hoodiehq/pouchdb-admins/pull/10)
- commented 6 times

[kmcrayton7](https://github.com/kmcrayton7) worked on [hoodiehq/editorial](https://github.com/hoodiehq/editorial)
- All-time total: 0 merged PRs.
- commented 2 times

[konson](https://github.com/konson) worked on [hoodiehq/hoodie-store-server](https://github.com/hoodiehq/hoodie-store-server)
- All-time total: 1 merged PRs.
- First PR on the hoodie project!
- landed [fix git clone URL](https://github.com/hoodiehq/hoodie-store-server/pull/37)
- commented 2 times

[kprenesti](https://github.com/kprenesti) worked on [hoodiehq/documentation](https://github.com/hoodiehq/documentation) and [hoodiehq/documentation](https://github.com/hoodiehq/documentation)
- All-time total: 1 merged PRs.
- First PR on the hoodie project!
- landed [Made edits to comply with style guide](https://github.com/hoodiehq/documentation/pull/204)
- commented on [hoodiehq/editorial#85](https://github.com/hoodiehq/editorial/issues/85#issuecomment-204763648)

[marisalaneous](https://github.com/marisalaneous) worked on 3 repositories.
- All-time total: 2 merged PRs.
- First PR on the hoodie project!
- landed [docs(README): fixed git clone URL](https://github.com/hoodiehq/hoodie-server-task/pull/22)
- landed [docs(README): setup instructions](https://github.com/hoodiehq/hoodie-account-client/pull/89)
- commented 7 times

[matsieftw](https://github.com/matsieftw) worked on [hoodiehq/editorial](https://github.com/hoodiehq/editorial)
- All-time total: 0 merged PRs.
- commented 2 times

[nickcolley](https://github.com/nickcolley) worked on 5 repositories.
- All-time total: 15 merged PRs.
- merged 2 pull requests
- reviewed 3 pull requests
- commented 8 times

[nolanlawson](https://github.com/nolanlawson) worked on [hoodiehq/hoodie-account-server](https://github.com/hoodiehq/hoodie-account-server)
- All-time total: 0 merged PRs.
- commented on [hoodiehq/hoodie-account-server#137](https://github.com/hoodiehq/hoodie-account-server/pull/137#issuecomment-204982603)

[pawan92](https://github.com/pawan92) worked on [hoodiehq/hood.ie](https://github.com/hoodiehq/hood.ie) and [hoodiehq/hood.ie](https://github.com/hoodiehq/hood.ie)
- All-time total: 1 merged PRs.
- First PR on the hoodie project!
- landed [added blog post with nicks interview](https://github.com/hoodiehq/hood.ie/pull/266)
- commented 11 times

[raquelxmoss](https://github.com/raquelxmoss) worked on [hoodiehq/documentation](https://github.com/hoodiehq/documentation) and [hoodiehq/documentation](https://github.com/hoodiehq/documentation)
- All-time total: 1 merged PRs.
- First PR on the hoodie project!
- landed [Proof-read Hoodie Tutorial](https://github.com/hoodiehq/documentation/pull/206)
- commented 3 times

[renrutnnej](https://github.com/renrutnnej) worked on 3 repositories.
- All-time total: 7 merged PRs.
- landed [Add TGIF for this week to the blog!](https://github.com/hoodiehq/hood.ie/pull/271)
- landed [Let's add Blog Bear to the pack!](https://github.com/hoodiehq/hood.ie/pull/273)
- merged 5 pull requests
- reviewed 5 pull requests
- commented 35 times

[roubaeli](https://github.com/roubaeli) worked on [hoodiehq/hoodie-account-server](https://github.com/hoodiehq/hoodie-account-server)
- All-time total: 1 merged PRs.
- First PR on the hoodie project!
- landed [test(routes): PUT /session/account 409 error message](https://github.com/hoodiehq/hoodie-account-server/pull/112)
- commented 2 times

[rubymorillo](https://github.com/rubymorillo) worked on [hoodiehq/hood.ie](https://github.com/hoodiehq/hood.ie) and [hoodiehq/hood.ie](https://github.com/hoodiehq/hood.ie)
- All-time total: 1 merged PRs.
- First PR on the hoodie project!
- landed [Added missing tags and added space in a phrase](https://github.com/hoodiehq/hood.ie/pull/267)
- commented 9 times

[sebbel](https://github.com/sebbel) worked on 5 repositories.
- All-time total: 2 merged PRs.
- landed [feat(remove-cookies): create plugin,tests and registration](https://github.com/hoodiehq/hoodie-account-server/pull/145)
- commented 14 times

[sergi](https://github.com/sergi) worked on [hoodiehq/hoodie-server](https://github.com/hoodiehq/hoodie-server)
- All-time total: 0 merged PRs.
- commented 2 times

[shyNotSly](https://github.com/shyNotSly) worked on [hoodiehq/hoodie-account](https://github.com/hoodiehq/hoodie-account)
- All-time total: 0 merged PRs.
- commented on [hoodiehq/hoodie-account#4](https://github.com/hoodiehq/hoodie-account/issues/4#issuecomment-204784129)

[sjnorth](https://github.com/sjnorth) worked on [hoodiehq/editorial](https://github.com/hoodiehq/editorial)
- All-time total: 2 merged PRs.
- commented on [hoodiehq/editorial#88](https://github.com/hoodiehq/editorial/issues/88#issuecomment-211769063)

[skeskali](https://github.com/skeskali) worked on [hoodiehq/hood.ie](https://github.com/hoodiehq/hood.ie)
- All-time total: 8 merged PRs.
- landed [Create 2016-04-08-tgif-tbd.md](https://github.com/hoodiehq/hood.ie/pull/272)
- landed [preparing TGIF 76](https://github.com/hoodiehq/hood.ie/pull/262)
- landed [Create 2016-04-15-tbd.md](https://github.com/hoodiehq/hood.ie/pull/275)

[stevemasta34](https://github.com/stevemasta34) worked on [hoodiehq/hoodie-account-server](https://github.com/hoodiehq/hoodie-account-server)
- All-time total: 1 merged PRs.
- commented 2 times

[summerspirit](https://github.com/summerspirit) worked on [hoodiehq/editorial](https://github.com/hoodiehq/editorial)
- All-time total: 0 merged PRs.
- commented 2 times

[texassiga](https://github.com/texassiga) worked on [hoodiehq/hoodie-account-server](https://github.com/hoodiehq/hoodie-account-server)
- All-time total: 0 merged PRs.
- commented 2 times

[timblack1](https://github.com/timblack1) worked on [hoodiehq/hoodie-server](https://github.com/hoodiehq/hoodie-server)
- All-time total: 2 merged PRs.
- landed [Added quote](https://github.com/hoodiehq/hoodie-server/pull/491)

[turcapat](https://github.com/turcapat) worked on [hoodiehq/editorial](https://github.com/hoodiehq/editorial)
- All-time total: 0 merged PRs.
- commented 10 times

[willholley](https://github.com/willholley) worked on [hoodiehq/faq](https://github.com/hoodiehq/faq)
- All-time total: 0 merged PRs.
- commented on [hoodiehq/faq#77](https://github.com/hoodiehq/faq/issues/77#issuecomment-203833426)

### PR DESCRIPTION
**Pull request by sjnorth**
closes https://github.com/hoodiehq/editorial/issues/88

### CODE REVIEW

**Comment by varjmes** [comment]
This. is. awesome.

**Comment by varjmes** [comment]
LGTM

File: _posts/2016-04-19-shoutouts-week13.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+* [Nolan](https://github.com/nolanlawson)
+* [Alex](https://github.com/espy)
+* [Pawan](https://github.com/pawan92)
+* [Stephanie](https://github.com/rubymorillo)
+* [Sergi](https://github.com/sergi)
+* [shyNotSly](https://github.com/shyNotSly)
+* [Stephen](https://github.com/stevemasta34)
+* [Summer](https://github.com/summerspirit)
+* [Texassiga](https://github.com/texassiga)
+* [Kim](https://github.com/kmcrayton7)
```
Comment (gr2m) [inline _posts/2016-04-19-shoutouts-week13.md]: I’d suggest we leave out the commenters this time, and for next time we try to prepare better data to review the comments? Would that be okay with every one?

Besides that, it’s great and good to merge from my side

**Comment by jennwrites** [comment]
Agree with @gr2m about the commenters because they could literally be randos not invested/interested in the project.

Otherwise LGTM

**Comment by gr2m** [comment]
I changed the "week 13" to "week 16". My idea was to use the current calendar week: http://whatweekisit.org/ – but we skipped 3 weeks since the last shout out. Merging it now, thanks @Charlotteis @renrutnnej  @sjnorth

**Pull request merged**